src/status_im2/contexts/communities/menus/community_options/__tests__/community_options_component_spec.cljs

```diff
@@ -0,0 +1,202 @@
+
+(ns status-im2.contexts.communities.menus.community-options.--tests--.community-options-component-spec
```

Comment:
From the guidelines and previous discussions, if the component test is isolated inside a single component directory, then we should name the file just `menus/community_options/component_spec.cljs`.

https://github.com/status-im/status-mobile/blob/8d08de3d4b736cbd1948ca4add07bc927aba8378/doc/new-guidelines.md#test-structure